packages/@lwc/style-compiler/src/transform.ts

```diff
@@ -15,14 +15,16 @@ export interface Config {
         /** Name of the module to resolve custom properties lookup */
         resolverModule?: string;
     };
+    scopeToken?: string;
```

Comment:
Add field description and update `README.md`.

Reply:
Done.